R/plot_taxa.R

```diff
@@ -1236,51 +1236,72 @@ plot_taxa_abund <- function(data,
 
 
 
-#' Plot sites on US map
+#' Work-in-progress upgraded plot_sites
 #'
 #' @param data (list or tbl_df, tbl, data.frame) The dataset object returned by \code{read_data()}, a named list of tables containing the observation and taxon tables, or a flat table containing columns of the observation and location tables.
 #' @param id (character) Identifier of dataset to be used in plot subtitles. Is automatically assigned when \code{data} is a dataset object containing the \code{id} field, or is a table containing the package_id column.
 #' @param alpha (numeric) Alpha-transparency scale of data points. Useful when many data points overlap. Allowed values are between 0 and 1, where 1 is 100\% opaque. Default is 1.
-#' @param labels (logical) Argument to show labels of each US state. Default is TRUE.
-#' 
+#' @param color_var (character) Argument to assign colors to the points on the plot
+#' @param shape_var (character) Argument to assign shapes to the points on the plot
+#' @param size_var (character) Argument to assign size to the points on the plot
+#'
 #' @return (gg, ggplot) A gg, ggplot object if assigned to a variable, otherwise a plot to your active graphics device
-#' 
+#'
 #' @details The \code{data} parameter accepts a range of input types but ultimately requires the 14 columns of the combined observation and location tables.
-#' 
+#'
 #' @export
-#' 
+#'
 #' @examples
 #' \dontrun{
+#'
+#'
+#'
 #' # Read a dataset of interest
 #' dataset <- read_data("edi.193.5")
-#' 
+#'
 #' # Plot the dataset
 #' plot_sites(dataset)
-#' 
+#'
 #' # Flatten dataset then plot
-#' dataset %>% 
-#'   flatten_data() %>% 
+#' dataset %>%
+#'   flatten_data() %>%
 #'   plot_sites()
 #' }
-#' 
+#'
+#'#Plot stacked datasets
+#'
+#'dataset2 <- read_data(ants_L1)
+#'
+#'flattened_data1 = flatten_data(dataset)
+#'flattened_data2 = flatten_data(dataset2)
+#'
+#'stacked_data = dplyr::bind_rows(dataset,dataset2)
+#'
+#'plot_sites(stacked_data)
+#'
+#'
 #' # Plot the example dataset
 #' plot_sites(ants_L1)
-#' 
-plot_sites <- function(data,
-                       id = NA_character_,
-                       alpha = 1,
-                       labels = TRUE) {
```

Comment:
@sokole @RafaelRangel0 Removing the `labels` parameter here breaks backwards compatibility. Any changes to backwards compatibility should be accompanied by deprecation warnings and the behavior phased out over the course of several months to allow users time to modify their applications.

For an example of the preferred deprecation approach, please see: https://github.com/EDIorg/ecocomDP/blob/6b56fa722af128e1702ead4dde79caa5b5d73a50/R/plot_taxa.R#L113